File: webSISGERT/ClientApp/src/app/ordenes-trabajo/detalle-repuesto/detalleRepuesto.ts
import { IRepuesto } from "../../repuestos/repuesto";
import { iOrdenTrabajo } from "../ordenTrabajo";

export interface IDetalleRepuesto {
  id: number,
  cantidad: number,
  precio: number,
  repuestoId: number,
  repuesto: IRepuesto,
  ordenTrabajoId: number,
  ordenTrabajo: iOrdenTrabajo
}

File: webSISGERT/Models/OT/DetalleRepuesto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace webSISGERT.Models.OT
{
    public class DetalleRepuesto
    {
        public int Id { get; set; }
        public int Cantidad { get; set; }
        public double Precio { get; set; }
        public int RepuestoId { get; set; }
        public virtual Repuesto repuesto { get; set; }
        public int OrdenTrabajoId { get; set; }
        public virtual OrdenTrabajo ordenTrabajo { get; set; }
    }
}

